File: Data/Models/CatalogoFilme.cs
namespace TrabalhoAPI.Data.Models
{
    public class CatalogoFilme
    {
        public int Id { get; set; }
        public int FilmeId { get; set; }
        public string? FilmeTitulo { get; set; }
        public Filme? Filme { get; set; }

        public int CatalogoId { get; set; }
        public Catalogo? Catalogo { get; set; }

    }
}

File: src/app/models/CatalogoFilme.ts
import { Catalogo } from "./Catalogo";
import { Filme } from "./Filme";

export class CatalogoFilme{
    id: number = 0;
    filmeId: number = 0;
    filmeTitulo: string = '';
    filme: Filme | undefined;
    catalogoId: number = 0;
    catalogo: Catalogo | undefined;
}
